play/settings/base.py

```diff
@@ -125,26 +113,15 @@ def is_production_env():
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
SQLite is a nice project, but you can't beat testing on the same database you'll be using in production.